### PR TITLE
chore: updated verida lib.

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- @verida/account-web-vault > 1.1.9
 - @verida/client-ts > 1.1.12
#102

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,3 @@
-PORT=3008
+REACT_APP_LOGO_URL=https://assets.verida.io/verida_login_request_logo_170x170.png
+REACT_APP_CONTEXT_NAME=Verida: Markdown Notes Demo
+REACT_APP_VERIDA_TESTNET_DEFAULT_SERVER=https://db.testnet.verida.io:5002
```
Comment (nick-verida) [inline .env:3]: Is this needed? Remove if not

**Review by nick-verida — CHANGES_REQUESTED** [review]
One minor change

File: .env (hunk 1 above)
Comment (cmcWebCode40) [reply .env]: i have removed unused env

**Review by nick-verida — APPROVED** [review]
looks good. Merging

**Pull request merged**